src/main/scala/it/unibo/authsim/library/dsl/UserProvider.scala

```diff
@@ -0,0 +1,32 @@
+package it.unibo.authsim.library.dsl
+
+import it.unibo.authsim.library.user.model.UserInformation
+
+/**
+ * Trait for a UserProvider
+ */
+trait UserProvider :
+  def userInformations(): List[UserInformation]
```

Comment:
There is no documentation for this method, one does not know how to implement it.